crates/turborepo-lib/src/run/summary/execution.rs

```diff
@@ -159,14 +168,23 @@ impl<'a> ExecutionSummary<'a> {
     fn successful(&self) -> usize {
         self.success + self.cached
     }
+
+    fn add_cached(&mut self, source: CacheSource) {
+        self.cached += 1;
```

Comment:
To preserve run summary output we just add a new field for remote rather than splitting into two. Naturally cached - cached_remote = cached_local if we need it one day.